[ORGSUM-23] Add speaker info page for Org Summit (#12889)
devopsdays/devopsdays-web: content/events/2023-organizer-summit/speaker-info.md

  - [Attendance](#attendance)
- [Travel and Lodging](#travel-and-lodging)
  - [Parking](#parking)
- [A/V](#av)
  - [Stage Details](#stage-details)
  - [Recording](#recording)
- [Contact Information](#contact-information)

<!-- /TOC -->

Thank you for being a part of our event! This document is our attempt to get as much useful information into one place as possible.

## Event Details and Schedule

### Venue Location

[Convene](https://convene.com/locations/chicago/233-south-wacker-drive/) (in the Willis Tower)

[233 S. Wacker Drive, Chicago, IL 60606](https://goo.gl/maps/MJASShEi1y7c3QNMA)

Directions [from ORD](https://goo.gl/maps/2rHBu57LMsCLbyacA) or [from MDW](https://goo.gl/maps/wzDJZnfNDw1nL19f7); parking [recommendations](https://spothero.com/search?kind=destination&id=400)

### Schedule

The event takes place on Tuesday, August 8, 2023. The detailed schedule for the day can be found at <https://devopsdays.org/events/2023-organizer-summit/program>.

A participants are invited to join for an evening reception/party Tuesday, August 8. Details are still being worked out, and more information will be shared as soon as available. Save the date!

### Check-in

> You'll be able to register from 8am on Tuesday, at the summit location.

Reviewer: ```suggestion
You'll be able to check in starting at 8am on Tuesday, at the summit location.

```

### Attendance

> We can learn so much from eachother! We hope to see you participate in all sessions, ask questions, suggest open spaces, during the Organizers Summit, as well as during Devopsdays Chicago.

Reviewer: ```suggestion
We can learn so much from each other! We hope to see you participate in all sessions, ask questions, suggest open spaces, during the Organizers Summit, as well as during Devopsdays Chicago.

```
